### PR TITLE
Handle AuthorizationError for killing jobs in cli

### LINKED ISSUE #1122: Error while killing a bunch of jobs
Case: `neuro kill job-A job-B job-C`. Suppose, my user has no permissions on `job-B`.

Expected behaviour:
```
job-A
ERROR: Not enough permissions ({"missing": [{"uri": "job-B", "action": "write"}]})
job-C
```
i.e., `neuro` failed to kill `job-B`, however it continued to kill other jobs (`job-C`).

Actual behaviour:

```
job-A
ERROR: Not enough permissions ({"missing": [{"uri": "job-B", "action": "write"}]})
```
i.e., `job-C` was not killed.

### PR DESCRIPTION
**Pull request by shagren**
closes #1122

### CODE REVIEW

File: neuromation/cli/job.py [hunk 1]
```diff
@@ -599,6 +600,8 @@ def job() -> None:
             click.echo(job_resolved)
         except ValueError as e:
             errors.append((job, e))
+        except AuthorizationError as e:
+            errors.append((job, ValueError(f"Not enough permissions")))
```
Comment (asvetlov) [inline neuromation/cli/job.py]: How to figure out what job has no permissions to be killed?

Comment (shagren) [reply neuromation/cli/job.py]: It's resolved few lines after:
```python
    def format_fail(job: str, reason: Exception) -> str:
        return click.style(f"Cannot kill job {job}: {reason}", fg="red")
```

Comment (shagren) [reply neuromation/cli/job.py]:
```
$ neuro job kill job-579a45da-1f1a-4328-8e14-116468764cce job-eeef0bf6-d30d-4bed-a894-ff34adf42e80
job-eeef0bf6-d30d-4bed-a894-ff34adf42e80
Cannot kill job job-579a45da-1f1a-4328-8e14-116468764cce: Not enough permissions

```

**Comment by shagren** [comment]
Change note added

**Pull request merged**